formulaire/src/main/resources/public/ts/controllers/forms-list.ts

```diff
@@ -766,6 +767,12 @@ export const formsListController = ng.controller('FormsListController', ['$scope
         vm.limitTable += vm.tableSize;
     };
 
+    vm.isCloseConfirmationOpen = () : boolean => {
+        let elem = document.getElementsByTagName('share-panel')[0].firstElementChild;
+        let sharePanelScope = angular.element(elem).scope();
```

Comment:
Tu peux le typer avec ": IScope"
```suggestion
        let sharePanelScope: IScope = angular.element(elem).scope();
```

Reply:
Si je type IScope il reconnait pas le '.display' que je cherche après